### ISSUE TITLE
d3.event is undefined (simple hello world example)

### ISSUE THREAD

**Issue by R-obert** [CCGE-BOADICEA/pedigreejs#117]
Hi,

I wanted to check out this software and I fail at a simple hello world example. My test folder looks like this:

![Screenshot from 2020-10-22 19-50-07](https://user-images.githubusercontent.com/62835651/96910859-5cd94d80-14a0-11eb-93fe-100e938ba754.png)

So basically I have just dumped everything into a folder: the needed libraries (1 folder for font-awesome, 1 folder for d3 and the jquery file), the pedigreejs folder, and my 2 custom files `pedigree_helloworld.html` and `pedigree_helloworld.js`. According to the [docs](https://ccge-boadicea.github.io/pedigreejs/docs/developer.html) that's all I need.

Content of **pedigreejs_helloworld.html**:

```
<!DOCTYPE html>

<html lang="en">
	<head>
		<meta charset="utf-8">

		<script src="jquery-3.5.1.min.js"></script>
		<script src="d3/d3.min.js"></script>
		<link rel="stylesheet" href="font-awesome-4.7.0/css/font-awesome.min.css">
	
		<link rel="stylesheet" href="pedigreejs-0.2/dist/css/pedigreejs.min.css">
		<script src="pedigreejs-0.2/dist/js/pedigreejs.min.js"></script>
	
		<script src="pedigree_helloworld.js"></script>
	</head>

	<body>
		<div id="ped_buttons"></div>
		<div id="ped_target"></div>
	</body>

</html>
```

Content of **pedigreejs_helloworld.js**:
```

$(document).ready(function(){
	var dataset = [ {"name": "m21", "sex": "M", "top_level": true}, {"name": "f21", "sex": "F", "top_level": true}, {"name": "ch1", "sex": "F", "mother": "f21", "father": "m21", "diabetes":  true, "proband": true}]

	var opts = {
    "targetDiv": "ped_target",
    "btn_target": "ped_buttons",
    "store_type": "array",
    "width": 1098,
    "height": 500,
    "symbol_size": 35,
    "edit": true,
    "diseases": [
        {
            "type": "diabetes",
            "colour": "#F68F35"
        }
    ],
    "DEBUG": false
	};
	var local_dataset = pedcache.current(opts);
	if (local_dataset !== undefined && local_dataset !== null) {
		opts.dataset = local_dataset;
	} else {
		opts.dataset = dataset;
	}
	opts= ptree.build(opts);
});
```

So basically I have just copied the dataset that I found on the website and tried to build a simple tree from it. Now, when I open the HTML in my browser, I see this:

![Screenshot from 2020-10-22 19-49-19](https://user-images.githubusercontent.com/62835651/96912020-f8b78900-14a1-11eb-94b7-adbfe552e16d.png)

Initially there are no errors in the console (i.e. after hitting F5). Then, as soon I move the mouse towards one of the nodes of the tree, the error messages appear in the console. The control buttons near the node just don't appear.

**Comment by tcarver** [comment]
Hi @R-obert what version of D3 are you using? It doesn't currently support v6. Try v5.16.0.

**Comment by R-obert** [comment]
... so simple :-) Thank you for the quick help!

**Comment by R-obert** [comment]
Okay I found out why: there is a necessary div with the id "node_properties". Without it the settings-button will have no effect in the GUI. Can you add that information (and mention jquery-ui) [here](https://ccge-boadicea.github.io/pedigreejs/docs/developer.html)? Unless I have overlooked something those 2 bits are missing from the docs.

**Comment by tcarver** [comment]
@R-obert thanks. I have updated the docs as requested. Also please note there is a new release - [v0.4](https://github.com/CCGE-BOADICEA/pedigreejs/releases).
